languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -59540,6 +59540,64 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
     <!-- Typografie -->
     <!-- ====================================================================== -->
     <category id="TYPOGRAPHY" name="Typografie">
+        <rulegroup id="SPACE_AFTER_BEFORE SLASH" name="Leerzeichen vor und nach Slash" default="temp_off"
+            <rule id="SPACE_AFTER_SLASH" name="Leerzeichen nach Slash">
+                <antipattern>
+                    <token regexp="yes">.*[\wüöäß].*</token>
+                    <token spacebefore="yes">/</token>
+                    <token regexp="yes" spacebefore="yes">.*[\wüöäß].*</token>
+                </antipattern>
+                <antipattern>
+                    <token regexp="yes">.*[\wüöäß].*</token>
+                    <token spacebefore="no">/</token>
+                    <token regexp="yes" spacebefore="no">.*[\wüöäß].*</token>
+                </antipattern>
+                <pattern>
+                    <marker>
+                        <token regexp="yes">.*[\wüöäß].*</token>
+                        <token spacebefore="no">/</token>
+                        <token regexp="yes" spacebefore="yes">.*[\wüöäß].*</token>
+                    </marker>
+                </pattern>
+                <message>Um Unregelmäßigkeiten zu vermeiden, wird empfohlen keine Leerzeichen vor und nach / zu verwenden. Alternativ kann auch jeweils ein Leerzeichen vor und nach / stehen.</message>
+                <suggestion>/</suggestion>
+                <suggestion> / </suggestion>
+                <example correction="">Ich ruf dich unter der Nummer <marker>010/ 5456846</marker> an.</example>
```

Comment:
1) Wenn du eine Suggestion hast, muss etwas in der Correction stehen.
2) Dieses Example stimmt mit deinem Pattern nicht überein.